rotkehlchen/tests/unit/test_pnl_debug.py

```diff
@@ -11,7 +11,7 @@ def test_pnl_debug_import(database):
     status, msg, data = json_importer.import_history_debug(pnl_debug_file)
     assert status is True
     assert msg == ''
-    assert len(data['events']) == 15
+    assert len(data['events']) == 17
```

Comment:
why less? Shouldn't the trades be replaced by history events?

Reply:
thy were replaced by swap events hence the extra two events.

Reply:
I don't understand what are the extra 2 events?

Reply:
the `pnl_debug.json` used to have two trade entries. each of those entry is now trade/spend & trade/receive swap events.